client/src/common/viame.service.js

```diff
@@ -54,6 +54,10 @@ function runImageConversion(folder) {
   );
 }
 
+function getPipelineList() {
+  return girderRest.get('viame/pipelines');
+}
+
```

Comment:
Is this used anywhere else but in the store? If not I think it can directly be part of the `fetchPipelines` action, unless you think it'll be used separately from the store (I guess if using the value in the store wouldn't make sense?) in the near future.

Reply:
The idea is to isolate `girderRest` into a single place in the code, mostly because our API is unstable and changes often.  I haven't gone through to do it everywhere, but I move it when it touches something I'm working on.